examples/agent_motion_prediction/agent_motion_prediction.ipynb

```diff
@@ -231,7 +228,11 @@
     "In this example, we simulate this pipeline by running `chop_dataset` on the validation set. The function stores:\n",
     "- a new chopped `.zarr` dataset, in which each scene has only the first 100 frames;\n",
     "- a numpy mask array where only valid agents in the 100th frame are True;\n",
-    "- a ground-truth file with the future coordinates of those agents;"
+    "- a ground-truth file with the future coordinates of those agents;\n",
+    "\n",
+    "Please note how `Num Frames` is equal to `Num Scenes*num_frames_to_chop`. \n",
```

Comment:
Minor: as @stefaniespeichert pointed out, I'd prefer Num Frames and num_frames_to_chop etc to have the same style.